plugins/plugin-core-support/src/lib/new-tab.ts

```diff
@@ -294,7 +324,7 @@ const newTab = async (basedOnEvent = false): Promise<boolean> => {
   }
 
   const currentVisibleTab = getCurrentTab()
-  currentVisibleTab['state'] = new TabState()
+  ;(currentVisibleTab['state'] as TabState).capture()
```

Comment:
My original code is 
```
const currentVisibleTab = getCurrentTab();
(currentVisibleTab['state'] as TabState).capture()
```

eslint or prettier changed this and added semicolon in front of all `as Type` guard .